src/GrumPHP/Task/Phpcs.php

```diff
@@ -84,4 +94,34 @@ public function run(ContextInterface $context)
 
         return TaskResult::createPassed($this, $context);
     }
+
+    /**
+     * @param array $config
+     *
+     * @return null|string
+     */
+    protected function getWhitelistPathPattern(array $config)
+    {
+        /** @var string|null $whitelistPathPattern */
+        $whitelistPathPattern = $config['whitelist_path_pattern'];
+        if (!$whitelistPathPattern) {
+            return null;
+        }
+
+        $whitelistPathPattern = $this->escapePatternDirectorySeparator($whitelistPathPattern);
+        $whitelistPathPattern = '/'.$whitelistPathPattern.'.(%s)$/';
```

Comment:
You can use the Regex util for this, it should also automatically escape strange characters.